buildenv/jenkins/common/pipeline-functions

```diff
@@ -527,4 +530,27 @@ def cleanup_artifactory(artifactory_manual_cleanup, job_name, artifactory_server
     }
 }
 
+def generate_test_jobs(TARGET_NAMES, SPEC){
+    target_names = TARGET_NAMES - EXCLUDED_TESTS
```

Comment:
I think excludes should match the exact target name so that is 100% clear what it going to be excluded.

Reply:
I'm not sure if we're on the same page or not but I think my thought was that jitaas is a test flag that can be passed to any test target rather than a test target itself.

Reply:
I made the change so that it checks the target name without the jitaas